packages/ui/src/accounts/components/AccountItem/components/RecoverButton.tsx

```diff
@@ -16,8 +16,6 @@ interface Props {
 export const RecoverButton = React.memo(({ memberId, lock, address, isRecoverable, isSmall }: Props) => {
   const { showModal } = useModal()
   const onClick = useCallback(() => {
```

Comment:
However "Voting" is the only lock that doesn't require a member id. I think the other ones will fail if it's not provided. And even though most of the time the member id should be there, if the locked staking account is present but the member controller account is not, the member id will be missing. Also it might be missing due to QN issues.

So here's what I'm proposing:
```suggestion
  const isVotingLock = lock.type === 'Voting'
  const onClick = useCallback(() => {
```

And `TransactionButton` should have a disabled state:
```tsx
<TransactionButton ... disabled={!isVoting && !memberId}
```